src/schema/nmdc.yaml

```diff
@@ -1499,10 +1488,6 @@ classes:
         description: Unique identifier for a bioproject submitted to INSDC that relates to the NMDC submitted study.
         annotations:
           display_hint: Provide the NCBI BioProject Accession Number associated with the listed NCBI BioProject Title.
-      #      emsl_project_identifier:
```

Comment:
Why are we removing this? I thought we just started sorting out using it
https://github.com/microbiomedata/nmdc-schema/issues/927

Reply:
unless it's somewhere else that I can't see in this review format